compiler/modules/types/common.bal

```diff
@@ -34,11 +34,15 @@ function memoSubtypeIsEmpty(Context cx, BddMemoTable memoTable, BddIsEmptyPredic
     BddMemo m;
```

Comment:
We should add a comment here explaining the values of MemoEmpty,

Reply:
fixed